src/colejorz/pilothouse.py

```diff
@@ -62,17 +65,17 @@ def _init_gpio(self):
         self.pwm.start(0)
         self.pwm_value = 0
 
-    def run_thread(self):
+    def run_thread(self) -> None:
         """Continuously monitor and apply speed change requests."""
         while not self._stop:
             self.event.wait()
             instruction = self._queue.get()
-            self.change_speed(**instruction)
+            self.change_speed(**instruction)  # type: ignore
```

Comment:
What if we have an int in the queue?

Reply:
I have changed `self._queue.put(0)` to `self._queue.put({'speed': 0, 'timed': 0})` in `StationMaster.change_state()` and updated the typing to avoid `TypeError` during runtime.